app/models/participant_declaration/npq.rb

```diff
@@ -1,12 +1,22 @@
 # frozen_string_literal: true
 
 class ParticipantDeclaration::NPQ < ParticipantDeclaration
-  belongs_to :participant_profile, class_name: "ParticipantProfile::NPQ"
+  QUALIFICATION_TYPES = {
+    "npq-leading-teaching" => "NPQLT",
+    "npq-leading-behaviour-culture" => "NPQLBC",
+    "npq-leading-teaching-development" => "NPQLTD",
+    "npq-leading-literacy" => "NPQLL",
+    "npq-senior-leadership" => "NPQSL",
+    "npq-headship" => "NPQH",
+    "npq-executive-leadership" => "NPQEL",
+    "npq-early-years-leadership" => "NPQEYL",
+    "npq-additional-support-offer" => "NPQASO",
+    "npq-early-headship-coaching-offer" => "NPQEHCO",
+  }.freeze
 
+  belongs_to :participant_profile, class_name: "ParticipantProfile::NPQ"
```

Comment:
just curious why we needed this as its in the parent class `ParticipantProfile` 🤔

Reply:
Hi @cwrw I don't why I had added this.. but seems all good without it.. tks 👍 😄